builders/cordova-build/schema.json

```diff
@@ -7,6 +7,15 @@
       "type": "string",
       "description": "Target to build."
     },
+    "consolelogs": {
+      "type": "boolean",
+      "description": "Inject script to print app console logs to Ionic CLI"
```

Comment:
Can we just say "Inject script to print console logs to the terminal"?

Reply:
Yes I thought this sounded weird/incorrect, but it's what the CLI currently says: https://github.com/ionic-team/ionic-cli/blob/6edf37367473cb3726cfefcd570e0a04b7fae461/packages/ionic/src/lib/project/ionic1/serve.ts#L24
https://github.com/ionic-team/ionic-cli/blob/6edf37367473cb3726cfefcd570e0a04b7fae461/packages/ionic/src/lib/project/ionic-angular/serve.ts#L32

I'll change it in the CLI PR as well: https://github.com/ionic-team/ionic-cli/pull/3900/files#diff-43bee07de60dee9e18ddd7a7fbc66673R42